### ISSUE TITLE
TypeError with exactOptionalPropertyTypes

### ISSUE THREAD

**Issue by marekdedic** [relative-ci/rollup-plugin-webpack-stats#325]
Hi, when I enable exactOptionalPropertyTypes in my tsconfig, I get the following type error:

```
node_modules/rollup-plugin-webpack-stats/dist/plugin.d.ts:3:11 - error TS2430: Interface 'WebpackStatsOptions' incorrectly extends interface 'BundleTransformOptions'.
  Types of property 'excludeAssets' are incompatible.
    Type 'ExcludeFilepathOption | undefined' is not assignable to type 'ExcludeFilepathOption'.
      Type 'undefined' is not assignable to type 'ExcludeFilepathOption'.

3 interface WebpackStatsOptions extends BundleTransformOptions {
            ~~~~~~~~~~~~~~~~~~~
```

**Comment by vio** [comment]
@marekdedic  Thank you for reporting the issue!

The plugin options type was wrongly extending internal types and the error surfaced once you set `exactOptionalPropertyTypes: true`.

I published  a fix on `v1.0.5-beta.0`. Can you please give it a try when you have the chance and let us know how it goes?

**Comment by marekdedic** [comment]
Hi, thanks for the fix, I can confirm that the beta works for me :)

**Comment by vio** [comment]
Thank you for checking! The fix is available on [`v1.0.5`](https://github.com/relative-ci/rollup-plugin-webpack-stats/releases/tag/v1.0.5) now ;)
